### PR TITLE
Add create and get single quiz and get all quizes

### PR DESCRIPTION
**Pull request by farah-najib**
This will only work locally for now until we finalize the domain and after the domain model change we will need to update it

Follow the comments to see how it works

quiz

### CODE REVIEW

File: serverless.yaml
Comment (ioannismarkou) [inline serverless.yaml:1]: region should be `eu-north-1`.

File: serverless.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            - http:
+                  path: quiz/{quizId}
+                  method: get
+    getAllQuizzes:
+        handler: handler.get_all_quizzes
+        events:
+            - http:
+                  path: quiz
+                  method: get
+
```
Comment (ioannismarkou) [inline serverless.yaml:39]: The table definition should be updated to the current one.

**Comment by ioannismarkou** [comment]
Good job on the functions! However, you should never merge a feature branch to the `main` branch but to  `develop`.

Screenshot 2024-09-26 at 20 54 33


Recommendations:
- You should close this pull request, pull all changes from dev after the other pull request is merged and then build these functions on top of that, to ensure that the serverless configuration is setup according to our plan.
- It's not necessary but good practice to create multiple commits in your branch, which makes it easier to revert specific changes if we need to. e.g. one commit per function, where you would both add the code in the handler and in the YAML file.

**Review by ioannismarkou — CHANGES_REQUESTED** [review]
Like I mentioned in the comment in the Conversation tab, you should close this pull request (shouldn't merge to main) and work on a new branch after we merge the other changes of the base serverless app.

**Comment by farah-najib** [comment]
> Good job on the functions! However, you should never merge a feature branch to the `main` branch but to `develop`.
> 
> Screenshot 2024-09-26 at 20 54 33
> Recommendations:
> 
> * You should close this pull request, pull all changes from dev after the other pull request is merged and then build these functions on top of that, to ensure that the serverless configuration is setup according to our plan.
> * It's not necessary but good practice to create multiple commits in your branch, which makes it easier to revert specific changes if we need to. e.g. one commit per function, where you would both add the code in the handler and in the YAML file.

When i created this branch there was no develop, who created this develop branch?

**Comment by ioannismarkou** [comment]
> When i created this branch there was no develop, who created this develop branch?

I did, and then I created a new feature branch from develop (`feature/serverless-project`)

**Pull request closed without merging**